examples/multisig/assets/sample-asc.teal

```diff
@@ -0,0 +1,8 @@
+// Check if Transaction type is pay, Sender is owner of Gold and amount is <= 100
+txn TypeEnum
```

Comment:
please update the comment as  per .teal example

Reply:
Ok